modules/drivers/mongo/src/metabase/driver/mongo/parameters.clj

```diff
@@ -107,7 +107,11 @@
     (format "{%s: %s}" (field->name field) (param-value->str field value))
     (substitute-one-field-filter field-filter)))
 
-(defn- substitute-param [param->value [acc missing] in-optional? {:keys [k], :as param}]
+(defn- substitute-native-query-snippet [[acc missing] v]
```

Comment:
Might even inline it...

Reply:
I think it's easier to read when it's in a separate line